pydm/widgets/slider.py

```diff
@@ -457,7 +462,7 @@ def userMinimum(self, new_min):
         ----------
         new_min : float
         """
-        self._user_minimum = float(new_min)
+        self._user_minimum = float(new_min) if new_min is not None else None
```

Comment:
What is the consequence of None in here at `self._user_minimum`?

Reply:
See my comment for self._user_maximum.